Tests/SourceKitLSPTests/CodeActionTests.swift

```diff
@@ -425,4 +425,54 @@ final class CodeActionTests: XCTestCase {
 
     try await fulfillmentOfOrThrow([editReceived])
   }
+
+  func testCodeActionForFixItsProducedBySwiftSyntax() async throws {
+    let ws = try await MultiFileTestWorkspace(files: [
+      "test.swift": "protocol Multi ident {}",
+      "compile_commands.json": "[]"
+    ])
+
+    let (uri, _) = try ws.openDocument("test.swift")
+
+    let report = try await ws.testClient.send(DocumentDiagnosticsRequest(textDocument: TextDocumentIdentifier(uri)))
+    guard case .full(let fullReport) = report else {
+      XCTFail("Expected full diagnostics report")
+      return
+    }
+
+    XCTAssertEqual(fullReport.items.count, 1)
+    let diagnostic = try XCTUnwrap(fullReport.items.first)
+    guard let codeActions = diagnostic.codeActions else {
+      XCTFail("Expected non-empty codeActions")
+      return
+    }
```

Comment:
You could just use `XCTUnwrap`

```suggestion
    let codeActions = try XCTUnwrap(diagnostic.codeAction)
```